spec/zoneddatetime.html

```diff
@@ -1399,5 +1354,42 @@ <h1>
           }.
       </emu-alg>
     </emu-clause>
+    <emu-clause id="sec-temporal-differencetemporalzoneddatetime" aoid="DifferenceTemporalZonedDateTime">
+      <h1>
+        DifferenceTemporalZonedDateTime (
+          _direction_: 1 or -1,
+          _zonedDateTime_: a Temporal.ZonedDateTime,
+          _other_: an ECMAScript language value,
+          _options_: an Object or *undefined*,
```

Comment:
```suggestion
          _options_: an ECMAScript language value,
```

Reply:
Why should we annotate options as an ECMAScript language value for some type but not the other types?